Eastern-Research-Group/expert-query
# Feature/115 etl national exports

## Related Issues:
* [EQ-115](https://jira.epa.gov/browse/EQ-115)

## Main Changes:
* Added the ability to etl national downloads to S3.
  * This uses the same rules as our schema retention.

## Steps To Test:
1. Run `npm run etl_database`
2. Verify the `app\server\app\content-etl\national-downloads\latest` folder was created
3. Verify the folder contains `csv`, `tsv`, `json` and `xlsx` versions of each profile
4. Repeat step 1 
5. Verify the `app\server\app\content-etl\national-downloads\<schema name>` folder was created
